wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/elevatorsimulation/Robot.java

```diff
@@ -104,13 +118,17 @@ public void simulationPeriodic() {
   public void teleopPeriodic() {
     if (m_joystick.getTrigger()) {
       // Here, we run PID control like normal, with a constant setpoint of 30in.
+      var setpoint = m_controller.getSetpoint();
       double pidOutput = m_controller.calculate(m_encoder.getDistance(), Units.inchesToMeters(30));
-      m_motor.setVoltage(pidOutput);
+      double feedforwardOutput = m_feedforward.calculate(setpoint.velocity);
+      m_motor.setVoltage(pidOutput + feedforwardOutput);
     } else {
       // Otherwise, we disable the motor.
       m_motor.set(0.0);
     }
   }
+  // To view the Elevator Sim in the simulator, select Network Tables -> SmartDashboard -> Elevator
+  // Sim
```

Comment:
The wrapping here is unfortunate, and differs between Java and C++. Is there anything we can do about it?

Reply:
removed the first Sim in the sentence so that all of it would fit in one line